spec/models/message_spec.rb

```diff
@@ -28,5 +29,36 @@
   describe 'validations' do
     it { is_expected.to validate_presence_of(:text) }
     it { is_expected.to belong_to(:conversation) }
+    it { is_expected.to validate_presence_of(:message_type) }
+  end
+
+  describe '#increment_unread' do
+    context 'when message is target match' do
+      let!(:message) do
+        create(:message, conversation: conversation,
```

Comment:
You can create a trait. so you can call `create(:message, :match, ...)` and you don't need to specify `message_type: 'match'` every time.